gwcli/storage.py

```diff
@@ -684,6 +686,32 @@ def __init__(self, parent, image_id, image_config, size=None,
         disk_map[self.image_id]['size'] = self.size
         disk_map[self.image_id]['size_h'] = self.size_h
 
+    def _apply_status(self):
+        disk_api = ('{}://localhost:{}/api/'
+                    'disk/{}'.format(self.http_mode,
+                                     settings.config.api_port, self.image_id))
+        self.logger.debug("disk GET status for {}".format(self.image_id))
+        api = APIRequest(disk_api)
+        api.get()
+
+        if api.response.status_code == 200:
+            info = api.response.json()
+            status = info.get("status")
+
+            self.__setattr__('current_owner', 'Unknown')
```

Comment:
Nit: move this up before the `status_code == 200` check

Reply:
At the same time I will rename the `current_owner` to `lock_owner`, which will be the same with the output from the curl GET. Or it will be a little confusing.